src/SUMMARY.md

```diff
@@ -25,6 +25,13 @@
     * [Accessing Glean data](concepts/glean/accessing_glean_data.md)
     * [Glean Debug ping viewer](concepts/glean/debug_ping_view.md)
   * [Growth & Usage Dashboard (GUD)](tools/gud.md)
+* [Metrics](metrics/index.md)
+  * [Overview](metrics/index.md)
```

Comment:
Do we need this Overview bullet; I assume the parent `[Metrics]` entry is where we'd give the overview. And we'd want to remove the `[Standard Metrics]` link in getting started or replace that with a separate doc that links to the Metrics section.

Reply:
Thanks.  I agree that the overview bullet is unnecessary.  And I did intent for the link in getting started to be a brief introduction with a link to this section.  Will make those changes now.